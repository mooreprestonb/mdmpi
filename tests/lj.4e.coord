<coords>
<natoms> 4 </natoms> <step> 0 </step>
<xyz> 0.999009 0     0           </xyz><vxyz> 10 0 0</vxyz><charge> 1 </charge>
<xyz> 2.99999  0   -9.08203e-06  </xyz><vxyz>0 0 0  </vxyz><charge>-1 </charge>
<xyz> 0.999    0     3           </xyz><vxyz>0 0 0  </vxyz><charge> 1 </charge>
<xyz> 3        0.001 -1.99999    </xyz><vxyz>0 0 0  </vxyz><charge>-1 </charge>
</coords>
