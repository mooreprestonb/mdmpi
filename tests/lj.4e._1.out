Getting initial energy and force
Starting MD ntime = 1000 with 12 dof
<step>        0 
<energyAVG>
<ham>  4.95649e+01 ~  4.95649e+01 </ham>
<ke>  5.00000e+01 ~  5.00000e+01 </ke> 
<pe> -4.35147e-01 ~ -4.35147e-01 </pe>
</energyAVG>
</step>
<step>       10 
<energyAVG>
<ham>  4.95185e+01 ~  4.91299e+01 </ham>
<ke>  5.04902e+01 ~  4.98601e+01 </ke> 
<pe> -9.71623e-01 ~ -7.30223e-01 </pe>
</energyAVG>
</step>
<step>       20 
<energyAVG>
<ham>  4.92502e+01 ~  4.90360e+01 </ham>
<ke>  4.79001e+01 ~  4.99786e+01 </ke> 
<pe>  1.35008e+00 ~ -9.42638e-01 </pe>
</energyAVG>
</step>
<step>       30 
<energyAVG>
<ham>  4.91802e+01 ~  4.90326e+01 </ham>
<ke>  4.84579e+01 ~  4.95014e+01 </ke> 
<pe>  7.22353e-01 ~ -4.68836e-01 </pe>
</energyAVG>
</step>
<step>       40 
<energyAVG>
<ham>  4.91832e+01 ~  4.92323e+01 </ham>
<ke>  4.87035e+01 ~  4.94545e+01 </ke> 
<pe>  4.79687e-01 ~ -2.22193e-01 </pe>
</energyAVG>
</step>
<step>       50 
<energyAVG>
<ham>  4.91928e+01 ~  4.92323e+01 </ham>
<ke>  4.88513e+01 ~  4.94629e+01 </ke> 
<pe>  3.41477e-01 ~ -2.30555e-01 </pe>
</energyAVG>
</step>
<step>       60 
<energyAVG>
<ham>  4.91662e+01 ~  4.88300e+01 </ham>
<ke>  4.89567e+01 ~  4.95412e+01 </ke> 
<pe>  2.09448e-01 ~ -7.11287e-01 </pe>
</energyAVG>
</step>
<step>       70 
<energyAVG>
<ham>  4.91301e+01 ~  4.90304e+01 </ham>
<ke>  4.90483e+01 ~  4.96594e+01 </ke> 
<pe>  8.18134e-02 ~ -6.29014e-01 </pe>
</energyAVG>
</step>
<step>       80 
<energyAVG>
<ham>  4.91178e+01 ~  4.90305e+01 </ham>
<ke>  4.91330e+01 ~  4.98087e+01 </ke> 
<pe> -1.52417e-02 ~ -7.78175e-01 </pe>
</energyAVG>
</step>
<step>       90 
<energyAVG>
<ham>  4.91083e+01 ~  4.90330e+01 </ham>
<ke>  4.92276e+01 ~  5.02619e+01 </ke> 
<pe> -1.19315e-01 ~ -1.22885e+00 </pe>
</energyAVG>
</step>
<step>      100 
<energyAVG>
<ham>  4.90903e+01 ~  4.82198e+01 </ham>
<ke>  4.89205e+01 ~  4.30889e+01 </ke> 
<pe>  1.69849e-01 ~  5.13092e+00 </pe>
</energyAVG>
</step>
<step>      110 
<energyAVG>
<ham>  4.90958e+01 ~  4.91960e+01 </ham>
<ke>  4.91323e+01 ~  5.13107e+01 </ke> 
<pe> -3.64961e-02 ~ -2.11468e+00 </pe>
</energyAVG>
</step>
<step>      120 
<energyAVG>
<ham>  4.90991e+01 ~  4.89960e+01 </ham>
<ke>  4.92937e+01 ~  5.06785e+01 </ke> 
<pe> -1.94567e-01 ~ -1.68253e+00 </pe>
</energyAVG>
</step>
<step>      130 
<energyAVG>
<ham>  4.90913e+01 ~  4.89960e+01 </ham>
<ke>  4.93642e+01 ~  4.99792e+01 </ke> 
<pe> -2.72934e-01 ~ -9.83221e-01 </pe>
</energyAVG>
</step>
<step>      140 
<energyAVG>
<ham>  4.90845e+01 ~  4.89957e+01 </ham>
<ke>  4.94000e+01 ~  4.98112e+01 </ke> 
<pe> -3.15527e-01 ~ -8.15495e-01 </pe>
</energyAVG>
</step>
<step>      150 
<energyAVG>
<ham>  4.90800e+01 ~  4.89957e+01 </ham>
<ke>  4.94260e+01 ~  4.97860e+01 </ke> 
<pe> -3.46043e-01 ~ -7.90283e-01 </pe>
</energyAVG>
</step>
<step>      160 
<energyAVG>
<ham>  4.90747e+01 ~  4.89957e+01 </ham>
<ke>  4.94479e+01 ~  4.97696e+01 </ke> 
<pe> -3.73173e-01 ~ -7.73888e-01 </pe>
</energyAVG>
</step>
<step>      170 
<energyAVG>
<ham>  4.90759e+01 ~  4.91944e+01 </ham>
<ke>  4.94657e+01 ~  4.97431e+01 </ke> 
<pe> -3.89840e-01 ~ -5.48744e-01 </pe>
</energyAVG>
</step>
<step>      180 
<energyAVG>
<ham>  4.90769e+01 ~  4.89934e+01 </ham>
<ke>  4.94803e+01 ~  4.97180e+01 </ke> 
<pe> -4.03362e-01 ~ -7.24542e-01 </pe>
</energyAVG>
</step>
<step>      190 
<energyAVG>
<ham>  4.90725e+01 ~  4.89934e+01 </ham>
<ke>  4.94920e+01 ~  4.96918e+01 </ke> 
<pe> -4.19448e-01 ~ -6.98344e-01 </pe>
</energyAVG>
</step>
<step>      200 
<energyAVG>
<ham>  4.90686e+01 ~  4.89934e+01 </ham>
<ke>  4.95011e+01 ~  4.96616e+01 </ke> 
<pe> -4.32511e-01 ~ -6.68184e-01 </pe>
</energyAVG>
</step>
<step>      210 
<energyAVG>
<ham>  4.90565e+01 ~  4.87937e+01 </ham>
<ke>  4.95076e+01 ~  4.96180e+01 </ke> 
<pe> -4.51108e-01 ~ -8.24285e-01 </pe>
</energyAVG>
</step>
<step>      220 
<energyAVG>
<ham>  4.90446e+01 ~  4.87937e+01 </ham>
<ke>  4.95114e+01 ~  4.95709e+01 </ke> 
<pe> -4.66824e-01 ~ -7.77197e-01 </pe>
</energyAVG>
</step>
<step>      230 
<energyAVG>
<ham>  4.90338e+01 ~  4.87937e+01 </ham>
<ke>  4.95129e+01 ~  4.95248e+01 </ke> 
<pe> -4.79153e-01 ~ -7.31061e-01 </pe>
</energyAVG>
</step>
<step>      240 
<energyAVG>
<ham>  4.90321e+01 ~  4.89942e+01 </ham>
<ke>  4.95128e+01 ~  4.95009e+01 </ke> 
<pe> -4.80705e-01 ~ -5.06658e-01 </pe>
</energyAVG>
</step>
<step>      250 
<energyAVG>
<ham>  4.90306e+01 ~  4.89942e+01 </ham>
<ke>  4.95119e+01 ~  4.94824e+01 </ke> 
<pe> -4.81322e-01 ~ -4.88199e-01 </pe>
</energyAVG>
</step>
<step>      260 
<energyAVG>
<ham>  4.90292e+01 ~  4.89942e+01 </ham>
<ke>  4.95105e+01 ~  4.94677e+01 </ke> 
<pe> -4.81263e-01 ~ -4.73429e-01 </pe>
</energyAVG>
</step>
<step>      270 
<energyAVG>
<ham>  4.90279e+01 ~  4.89942e+01 </ham>
<ke>  4.95087e+01 ~  4.94560e+01 </ke> 
<pe> -4.80728e-01 ~ -4.61733e-01 </pe>
</energyAVG>
</step>
<step>      280 
<energyAVG>
<ham>  4.90303e+01 ~  4.91941e+01 </ham>
<ke>  4.95067e+01 ~  4.94533e+01 </ke> 
<pe> -4.76376e-01 ~ -2.59276e-01 </pe>
</energyAVG>
</step>
<step>      290 
<energyAVG>
<ham>  4.90359e+01 ~  4.91941e+01 </ham>
<ke>  4.95049e+01 ~  4.94596e+01 </ke> 
<pe> -4.69032e-01 ~ -2.65489e-01 </pe>
</energyAVG>
</step>
<step>      300 
<energyAVG>
<ham>  4.90412e+01 ~  4.91941e+01 </ham>
<ke>  4.95036e+01 ~  4.94662e+01 </ke> 
<pe> -4.62389e-01 ~ -2.72115e-01 </pe>
</energyAVG>
</step>
<step>      310 
<energyAVG>
<ham>  4.90461e+01 ~  4.91941e+01 </ham>
<ke>  4.95025e+01 ~  4.94733e+01 </ke> 
<pe> -4.56395e-01 ~ -2.79200e-01 </pe>
</energyAVG>
</step>
<step>      320 
<energyAVG>
<ham>  4.90507e+01 ~  4.91941e+01 </ham>
<ke>  4.95017e+01 ~  4.94809e+01 </ke> 
<pe> -4.51004e-01 ~ -2.86802e-01 </pe>
</energyAVG>
</step>
<step>      330 
<energyAVG>
<ham>  4.90550e+01 ~  4.91941e+01 </ham>
<ke>  4.95012e+01 ~  4.94891e+01 </ke> 
<pe> -4.46178e-01 ~ -2.94987e-01 </pe>
</energyAVG>
</step>
<step>      340 
<energyAVG>
<ham>  4.90591e+01 ~  4.91941e+01 </ham>
<ke>  4.95010e+01 ~  4.94979e+01 </ke> 
<pe> -4.41885e-01 ~ -3.03835e-01 </pe>
</energyAVG>
</step>
<step>      350 
<energyAVG>
<ham>  4.90629e+01 ~  4.91941e+01 </ham>
<ke>  4.95010e+01 ~  4.95075e+01 </ke> 
<pe> -4.38101e-01 ~ -3.13444e-01 </pe>
</energyAVG>
</step>
<step>      360 
<energyAVG>
<ham>  4.90666e+01 ~  4.91941e+01 </ham>
<ke>  4.95014e+01 ~  4.95180e+01 </ke> 
<pe> -4.34805e-01 ~ -3.23934e-01 </pe>
</energyAVG>
</step>
<step>      370 
<energyAVG>
<ham>  4.90700e+01 ~  4.91941e+01 </ham>
<ke>  4.95020e+01 ~  4.95295e+01 </ke> 
<pe> -4.31985e-01 ~ -3.35453e-01 </pe>
</energyAVG>
</step>
<step>      380 
<energyAVG>
<ham>  4.90733e+01 ~  4.91941e+01 </ham>
<ke>  4.95029e+01 ~  4.95423e+01 </ke> 
<pe> -4.29632e-01 ~ -3.48192e-01 </pe>
</energyAVG>
</step>
<step>      390 
<energyAVG>
<ham>  4.90764e+01 ~  4.91941e+01 </ham>
<ke>  4.95041e+01 ~  4.95565e+01 </ke> 
<pe> -4.27746e-01 ~ -3.62399e-01 </pe>
</energyAVG>
</step>
<step>      400 
<energyAVG>
<ham>  4.90793e+01 ~  4.91941e+01 </ham>
<ke>  4.95056e+01 ~  4.95725e+01 </ke> 
<pe> -4.26332e-01 ~ -3.78400e-01 </pe>
</energyAVG>
</step>
<step>      410 
<energyAVG>
<ham>  4.90821e+01 ~  4.91941e+01 </ham>
<ke>  4.95075e+01 ~  4.95907e+01 </ke> 
<pe> -4.25405e-01 ~ -3.96644e-01 </pe>
</energyAVG>
</step>
<step>      420 
<energyAVG>
<ham>  4.90847e+01 ~  4.91941e+01 </ham>
<ke>  4.95097e+01 ~  4.96118e+01 </ke> 
<pe> -4.24991e-01 ~ -4.17759e-01 </pe>
</energyAVG>
</step>
<step>      430 
<energyAVG>
<ham>  4.90873e+01 ~  4.91941e+01 </ham>
<ke>  4.95124e+01 ~  4.96367e+01 </ke> 
<pe> -4.25133e-01 ~ -4.42665e-01 </pe>
</energyAVG>
</step>
<step>      440 
<energyAVG>
<ham>  4.90897e+01 ~  4.91941e+01 </ham>
<ke>  4.95156e+01 ~  4.96668e+01 </ke> 
<pe> -4.25894e-01 ~ -4.72766e-01 </pe>
</energyAVG>
</step>
<step>      450 
<energyAVG>
<ham>  4.90920e+01 ~  4.91941e+01 </ham>
<ke>  4.95194e+01 ~  4.97044e+01 </ke> 
<pe> -4.27375e-01 ~ -5.10321e-01 </pe>
</energyAVG>
</step>
<step>      460 
<energyAVG>
<ham>  4.90942e+01 ~  4.91941e+01 </ham>
<ke>  4.95240e+01 ~  4.97533e+01 </ke> 
<pe> -4.29732e-01 ~ -5.59202e-01 </pe>
</energyAVG>
</step>
<step>      470 
<energyAVG>
<ham>  4.90964e+01 ~  4.91941e+01 </ham>
<ke>  4.95296e+01 ~  4.98207e+01 </ke> 
<pe> -4.33226e-01 ~ -6.26607e-01 </pe>
</energyAVG>
</step>
<step>      480 
<energyAVG>
<ham>  4.90984e+01 ~  4.91941e+01 </ham>
<ke>  4.95367e+01 ~  4.99214e+01 </ke> 
<pe> -4.38321e-01 ~ -7.27331e-01 </pe>
</energyAVG>
</step>
<step>      490 
<energyAVG>
<ham>  4.91003e+01 ~  4.91942e+01 </ham>
<ke>  4.95463e+01 ~  5.00901e+01 </ke> 
<pe> -4.45932e-01 ~ -8.95950e-01 </pe>
</energyAVG>
</step>
<step>      500 
<energyAVG>
<ham>  4.91022e+01 ~  4.91943e+01 </ham>
<ke>  4.95603e+01 ~  5.04190e+01 </ke> 
<pe> -4.58125e-01 ~ -1.22472e+00 </pe>
</energyAVG>
</step>
<step>      510 
<energyAVG>
<ham>  4.91040e+01 ~  4.91925e+01 </ham>
<ke>  4.95835e+01 ~  5.10405e+01 </ke> 
<pe> -4.79542e-01 ~ -1.84798e+00 </pe>
</energyAVG>
</step>
<step>      520 
<energyAVG>
<ham>  4.91057e+01 ~  4.92015e+01 </ham>
<ke>  4.95993e+01 ~  4.96621e+01 </ke> 
<pe> -4.93641e-01 ~ -4.60569e-01 </pe>
</energyAVG>
</step>
<step>      530 
<energyAVG>
<ham>  4.91073e+01 ~  4.91937e+01 </ham>
<ke>  4.96219e+01 ~  5.09242e+01 </ke> 
<pe> -5.14580e-01 ~ -1.73048e+00 </pe>
</energyAVG>
</step>
<step>      540 
<energyAVG>
<ham>  4.91089e+01 ~  4.91943e+01 </ham>
<ke>  4.96394e+01 ~  5.03373e+01 </ke> 
<pe> -5.30544e-01 ~ -1.14297e+00 </pe>
</energyAVG>
</step>
<step>      550 
<energyAVG>
<ham>  4.91104e+01 ~  4.91942e+01 </ham>
<ke>  4.96489e+01 ~  5.00505e+01 </ke> 
<pe> -5.38493e-01 ~ -8.56326e-01 </pe>
</energyAVG>
</step>
<step>      560 
<energyAVG>
<ham>  4.91119e+01 ~  4.91941e+01 </ham>
<ke>  4.96545e+01 ~  4.98989e+01 </ke> 
<pe> -5.42551e-01 ~ -7.04808e-01 </pe>
</energyAVG>
</step>
<step>      570 
<energyAVG>
<ham>  4.91134e+01 ~  4.91941e+01 </ham>
<ke>  4.96578e+01 ~  4.98062e+01 </ke> 
<pe> -5.44442e-01 ~ -6.12070e-01 </pe>
</energyAVG>
</step>
<step>      580 
<energyAVG>
<ham>  4.91148e+01 ~  4.91941e+01 </ham>
<ke>  4.96597e+01 ~  4.97430e+01 </ke> 
<pe> -5.44978e-01 ~ -5.48933e-01 </pe>
</energyAVG>
</step>
<step>      590 
<energyAVG>
<ham>  4.91161e+01 ~  4.91941e+01 </ham>
<ke>  4.96607e+01 ~  4.96967e+01 </ke> 
<pe> -5.44595e-01 ~ -5.02582e-01 </pe>
</energyAVG>
</step>
<step>      600 
<energyAVG>
<ham>  4.91174e+01 ~  4.91941e+01 </ham>
<ke>  4.96610e+01 ~  4.96607e+01 </ke> 
<pe> -5.43556e-01 ~ -4.66653e-01 </pe>
</energyAVG>
</step>
<step>      610 
<energyAVG>
<ham>  4.91187e+01 ~  4.91941e+01 </ham>
<ke>  4.96607e+01 ~  4.96317e+01 </ke> 
<pe> -5.42029e-01 ~ -4.37662e-01 </pe>
</energyAVG>
</step>
<step>      620 
<energyAVG>
<ham>  4.91199e+01 ~  4.91941e+01 </ham>
<ke>  4.96600e+01 ~  4.96076e+01 </ke> 
<pe> -5.40129e-01 ~ -4.13554e-01 </pe>
</energyAVG>
</step>
<step>      630 
<energyAVG>
<ham>  4.91210e+01 ~  4.91941e+01 </ham>
<ke>  4.96590e+01 ~  4.95871e+01 </ke> 
<pe> -5.37940e-01 ~ -3.93035e-01 </pe>
</energyAVG>
</step>
<step>      640 
<energyAVG>
<ham>  4.91222e+01 ~  4.91941e+01 </ham>
<ke>  4.96577e+01 ~  4.95693e+01 </ke> 
<pe> -5.35524e-01 ~ -3.75252e-01 </pe>
</energyAVG>
</step>
<step>      650 
<energyAVG>
<ham>  4.91233e+01 ~  4.91941e+01 </ham>
<ke>  4.96562e+01 ~  4.95537e+01 </ke> 
<pe> -5.32927e-01 ~ -3.59616e-01 </pe>
</energyAVG>
</step>
<step>      660 
<energyAVG>
<ham>  4.91244e+01 ~  4.91941e+01 </ham>
<ke>  4.96545e+01 ~  4.95398e+01 </ke> 
<pe> -5.30188e-01 ~ -3.45706e-01 </pe>
</energyAVG>
</step>
<step>      670 
<energyAVG>
<ham>  4.91254e+01 ~  4.91941e+01 </ham>
<ke>  4.96527e+01 ~  4.95273e+01 </ke> 
<pe> -5.27334e-01 ~ -3.33212e-01 </pe>
</energyAVG>
</step>
<step>      680 
<energyAVG>
<ham>  4.91264e+01 ~  4.91941e+01 </ham>
<ke>  4.96508e+01 ~  4.95160e+01 </ke> 
<pe> -5.24391e-01 ~ -3.21898e-01 </pe>
</energyAVG>
</step>
<step>      690 
<energyAVG>
<ham>  4.91274e+01 ~  4.91941e+01 </ham>
<ke>  4.96488e+01 ~  4.95056e+01 </ke> 
<pe> -5.21378e-01 ~ -3.11584e-01 </pe>
</energyAVG>
</step>
<step>      700 
<energyAVG>
<ham>  4.91283e+01 ~  4.91941e+01 </ham>
<ke>  4.96466e+01 ~  4.94962e+01 </ke> 
<pe> -5.18310e-01 ~ -3.02126e-01 </pe>
</energyAVG>
</step>
<step>      710 
<energyAVG>
<ham>  4.91293e+01 ~  4.91941e+01 </ham>
<ke>  4.96445e+01 ~  4.94875e+01 </ke> 
<pe> -5.15201e-01 ~ -2.93408e-01 </pe>
</energyAVG>
</step>
<step>      720 
<energyAVG>
<ham>  4.91302e+01 ~  4.91941e+01 </ham>
<ke>  4.96422e+01 ~  4.94794e+01 </ke> 
<pe> -5.12062e-01 ~ -2.85338e-01 </pe>
</energyAVG>
</step>
<step>      730 
<energyAVG>
<ham>  4.91310e+01 ~  4.91941e+01 </ham>
<ke>  4.96399e+01 ~  4.94719e+01 </ke> 
<pe> -5.08904e-01 ~ -2.77837e-01 </pe>
</energyAVG>
</step>
<step>      740 
<energyAVG>
<ham>  4.91319e+01 ~  4.91941e+01 </ham>
<ke>  4.96376e+01 ~  4.94649e+01 </ke> 
<pe> -5.05733e-01 ~ -2.70842e-01 </pe>
</energyAVG>
</step>
<step>      750 
<energyAVG>
<ham>  4.91327e+01 ~  4.91941e+01 </ham>
<ke>  4.96353e+01 ~  4.94584e+01 </ke> 
<pe> -5.02557e-01 ~ -2.64297e-01 </pe>
</energyAVG>
</step>
<step>      760 
<energyAVG>
<ham>  4.91335e+01 ~  4.91941e+01 </ham>
<ke>  4.96329e+01 ~  4.94522e+01 </ke> 
<pe> -4.99381e-01 ~ -2.58157e-01 </pe>
</energyAVG>
</step>
<step>      770 
<energyAVG>
<ham>  4.91343e+01 ~  4.91941e+01 </ham>
<ke>  4.96305e+01 ~  4.94464e+01 </ke> 
<pe> -4.96211e-01 ~ -2.52381e-01 </pe>
</energyAVG>
</step>
<step>      780 
<energyAVG>
<ham>  4.91351e+01 ~  4.91941e+01 </ham>
<ke>  4.96281e+01 ~  4.94410e+01 </ke> 
<pe> -4.93050e-01 ~ -2.46935e-01 </pe>
</energyAVG>
</step>
<step>      790 
<energyAVG>
<ham>  4.91358e+01 ~  4.91941e+01 </ham>
<ke>  4.96257e+01 ~  4.94359e+01 </ke> 
<pe> -4.89903e-01 ~ -2.41789e-01 </pe>
</energyAVG>
</step>
<step>      800 
<energyAVG>
<ham>  4.91365e+01 ~  4.91941e+01 </ham>
<ke>  4.96233e+01 ~  4.94310e+01 </ke> 
<pe> -4.86771e-01 ~ -2.36918e-01 </pe>
</energyAVG>
</step>
<step>      810 
<energyAVG>
<ham>  4.91373e+01 ~  4.91941e+01 </ham>
<ke>  4.96209e+01 ~  4.94264e+01 </ke> 
<pe> -4.83659e-01 ~ -2.32297e-01 </pe>
</energyAVG>
</step>
<step>      820 
<energyAVG>
<ham>  4.91379e+01 ~  4.91941e+01 </ham>
<ke>  4.96185e+01 ~  4.94220e+01 </ke> 
<pe> -4.80568e-01 ~ -2.27906e-01 </pe>
</energyAVG>
</step>
<step>      830 
<energyAVG>
<ham>  4.91386e+01 ~  4.91941e+01 </ham>
<ke>  4.96161e+01 ~  4.94178e+01 </ke> 
<pe> -4.77499e-01 ~ -2.23728e-01 </pe>
</energyAVG>
</step>
<step>      840 
<energyAVG>
<ham>  4.91393e+01 ~  4.91941e+01 </ham>
<ke>  4.96137e+01 ~  4.94138e+01 </ke> 
<pe> -4.74456e-01 ~ -2.19746e-01 </pe>
</energyAVG>
</step>
<step>      850 
<energyAVG>
<ham>  4.91399e+01 ~  4.91941e+01 </ham>
<ke>  4.96114e+01 ~  4.94100e+01 </ke> 
<pe> -4.71438e-01 ~ -2.15946e-01 </pe>
</energyAVG>
</step>
<step>      860 
<energyAVG>
<ham>  4.91406e+01 ~  4.91941e+01 </ham>
<ke>  4.96090e+01 ~  4.94064e+01 </ke> 
<pe> -4.68447e-01 ~ -2.12314e-01 </pe>
</energyAVG>
</step>
<step>      870 
<energyAVG>
<ham>  4.91412e+01 ~  4.91941e+01 </ham>
<ke>  4.96066e+01 ~  4.94029e+01 </ke> 
<pe> -4.65484e-01 ~ -2.08840e-01 </pe>
</energyAVG>
</step>
<step>      880 
<energyAVG>
<ham>  4.91418e+01 ~  4.91941e+01 </ham>
<ke>  4.96043e+01 ~  4.93996e+01 </ke> 
<pe> -4.62550e-01 ~ -2.05511e-01 </pe>
</energyAVG>
</step>
<step>      890 
<energyAVG>
<ham>  4.91424e+01 ~  4.91941e+01 </ham>
<ke>  4.96020e+01 ~  4.93964e+01 </ke> 
<pe> -4.59646e-01 ~ -2.02319e-01 </pe>
</energyAVG>
</step>
<step>      900 
<energyAVG>
<ham>  4.91438e+01 ~  4.93944e+01 </ham>
<ke>  4.95997e+01 ~  4.93944e+01 </ke> 
<pe> -4.55884e-01 ~  0.00000e+00 </pe>
</energyAVG>
</step>
<step>      910 
<energyAVG>
<ham>  4.91463e+01 ~  4.91941e+01 </ham>
<ke>  4.95974e+01 ~  4.93945e+01 </ke> 
<pe> -4.51100e-01 ~ -2.00405e-01 </pe>
</energyAVG>
</step>
<step>      920 
<energyAVG>
<ham>  4.91469e+01 ~  4.91941e+01 </ham>
<ke>  4.95953e+01 ~  4.93973e+01 </ke> 
<pe> -4.48394e-01 ~ -2.03152e-01 </pe>
</energyAVG>
</step>
<step>      930 
<energyAVG>
<ham>  4.91474e+01 ~  4.91941e+01 </ham>
<ke>  4.95931e+01 ~  4.94001e+01 </ke> 
<pe> -4.45777e-01 ~ -2.05996e-01 </pe>
</energyAVG>
</step>
<step>      940 
<energyAVG>
<ham>  4.91479e+01 ~  4.91941e+01 </ham>
<ke>  4.95911e+01 ~  4.94030e+01 </ke> 
<pe> -4.43246e-01 ~ -2.08943e-01 </pe>
</energyAVG>
</step>
<step>      950 
<energyAVG>
<ham>  4.91484e+01 ~  4.91941e+01 </ham>
<ke>  4.95892e+01 ~  4.94061e+01 </ke> 
<pe> -4.40800e-01 ~ -2.11999e-01 </pe>
</energyAVG>
</step>
<step>      960 
<energyAVG>
<ham>  4.91488e+01 ~  4.91941e+01 </ham>
<ke>  4.95873e+01 ~  4.94093e+01 </ke> 
<pe> -4.38437e-01 ~ -2.15171e-01 </pe>
</energyAVG>
</step>
<step>      970 
<energyAVG>
<ham>  4.91493e+01 ~  4.91941e+01 </ham>
<ke>  4.95855e+01 ~  4.94126e+01 </ke> 
<pe> -4.36156e-01 ~ -2.18465e-01 </pe>
</energyAVG>
</step>
<step>      980 
<energyAVG>
<ham>  4.91498e+01 ~  4.91941e+01 </ham>
<ke>  4.95837e+01 ~  4.94160e+01 </ke> 
<pe> -4.33956e-01 ~ -2.21890e-01 </pe>
</energyAVG>
</step>
<step>      990 
<energyAVG>
<ham>  4.91502e+01 ~  4.91941e+01 </ham>
<ke>  4.95820e+01 ~  4.94196e+01 </ke> 
<pe> -4.31836e-01 ~ -2.25453e-01 </pe>
</energyAVG>
</step>
<step>     1000 
<energyAVG>
<ham>  4.91506e+01 ~  4.91941e+01 </ham>
<ke>  4.95804e+01 ~  4.94233e+01 </ke> 
<pe> -4.29794e-01 ~ -2.29163e-01 </pe>
</energyAVG>
</step>

